language/move-prover/tests/sources/functional/aborts_if.cvc4_exp

```diff
@@ -1,181 +1,51 @@
-     │
- 145 │ ╭     fun abort_at_2_or_3_spec_incorrect(x: u64) {
- 146 │ │         if (x == 2 || x == 3) abort 1;
- 147 │ │     }
-     │ ╰─────^
-     │
-
-error: verification inconclusive
-
-     ┌── tests/sources/functional/aborts_if.move:154:5 ───
-     │
- 154 │ ╭     fun abort_at_2_or_3_strict_incorrect(x: u64) {
- 155 │ │         if (x == 2 || x == 3) abort 1;
- 156 │ │     }
-     │ ╰─────^
-     │
-
-error: verification inconclusive
-
-     ┌── tests/sources/functional/aborts_if.move:136:5 ───
-     │
- 136 │ ╭     fun abort_at_2_or_3_total_incorrect(x: u64) {
- 137 │ │         if (x == 2 || x == 3) abort 1;
- 138 │ │     }
-     │ ╰─────^
-     │
-
-error: verification inconclusive
-
-     ┌── tests/sources/functional/aborts_if.move:170:5 ───
-     │
- 170 │ ╭     fun aborts_if_with_code(x: u64) {
- 171 │ │         if (x == 2 || x == 3) abort_1();
- 172 │ │     }
-     │ ╰─────^
-     │
-
-error: verification inconclusive
-
-    ┌── tests/sources/functional/aborts_if.move:77:5 ───
-    │
- 77 │ ╭     fun multi_abort1(x: u64, y: u64) {
- 78 │ │         if (x <= y) abort 1
- 79 │ │     }
-    │ ╰─────^
-    │
-
-error: verification inconclusive
-
-    ┌── tests/sources/functional/aborts_if.move:86:5 ───
-    │
- 86 │ ╭     fun multi_abort2_incorrect(x: u64, y: u64) {
- 87 │ │         if (x < y) abort 1
- 88 │ │     }
-    │ ╰─────^
-    │
-
-error: verification inconclusive
-
-    ┌── tests/sources/functional/aborts_if.move:95:5 ───
-    │
- 95 │ ╭     fun multi_abort3_incorrect(_x: u64, _y: u64) {
- 96 │ │         abort 1
- 97 │ │     }
-    │ ╰─────^
-    │
-
-error: verification inconclusive
-
-     ┌── tests/sources/functional/aborts_if.move:104:5 ───
-     │
- 104 │ ╭     fun multi_abort4(_x: u64, _y: u64) {
- 105 │ │         abort 1
- 106 │ │     }
-     │ ╰─────^
-     │
-
-error: verification inconclusive
-
-     ┌── tests/sources/functional/aborts_if.move:113:5 ───
-     │
- 113 │ ╭     fun multi_abort5_incorrect(x: u64) {
- 114 │ │         if (x == 0) {
- 115 │ │             abort 1
- 116 │ │         };
- 117 │ │     }
-     │ ╰─────^
-     │
-
-error: verification inconclusive
-
-    ┌── tests/sources/functional/aborts_if.move:12:5 ───
-    │
- 12 │ ╭     fun no_aborts_if(_x: u64, _y: u64) {
- 13 │ │         abort 1
- 14 │ │     }
-    │ ╰─────^
-    │
+Move prover returns: [internal] boogie exited with prover errors:
+Prover error: Unexpected value: $Error
+
+Advisory: $TestAbortsIf_abort2_incorrect$verify SKIPPED because of internal error: unexpected prover output: Exception of type 'Microsoft.Boogie.SMTLib.SMTLibProcessTheoremProver+BadExprFromProver' was thrown.
+aborts_if.bpl(1785,27): Verification inconclusive ($TestAbortsIf_abort2_incorrect$verify)
+aborts_if.bpl(1830,27): Verification inconclusive ($TestAbortsIf_abort3$verify)
+aborts_if.bpl(1886,27): Verification inconclusive ($TestAbortsIf_abort4_incorrect$verify)
+Prover error: Invalid PING response from the prover: unsat
```

Comment:
What about just disabling these tests from with the CVC4 feature instead of recording these error messages in the exp file?

Reply:
I think it's fine to keep these tests. This should be fixed soon, and then the baseline can be regenerated.